app/models/katello/content_view_version.rb

```diff
@@ -266,45 +266,45 @@ def docker_tag_count
     end
 
     def debs
-      Katello::Deb.in_repositories(self.repositories.archived).uniq
+      Katello::Deb.in_repositories(self.repositories.archived).distinct
     end
 
     def deb_count
       debs.count
     end
 
     def errata(errata_type = nil)
-      errata = Erratum.in_repositories(archived_repos).uniq
+      errata = Erratum.in_repositories(archived_repos).distinct
       errata = errata.of_type(errata_type) if errata_type
       errata
     end
 
     def library_errata
-      Erratum.in_repositories(library_repos).uniq
+      Erratum.in_repositories(library_repos).distinct
     end
 
     def available_errata
       library_errata.where.not(:id => errata)
     end
 
     def file_units
-      FileUnit.in_repositories(archived_repos).uniq
+      FileUnit.in_repositories(archived_repos).distinct
     end
 
     def ostree_branches
-      OstreeBranch.in_repositories(archived_repos).uniq
+      OstreeBranch.in_repositories(archived_repos).distinct
     end
 
     def docker_manifests
-      DockerManifest.in_repositories(archived_repos).uniq
+      DockerManifest.in_repositories(archived_repos).distinct
     end
 
     def docker_manifest_lists
-      DockerManifestList.in_repositories(archived_repos).uniq
+      DockerManifestList.in_repositories(archived_repos).distinct
     end
 
     def package_groups
-      PackageGroup.in_repositories(archived_repos).uniq
+      PackageGroup.in_repositories(archived_repos).distinct
```

Comment:
I think i wrote a test that ensure in_repositories returns distinct values already (since thats all anyone would ever want).  I don't think this would be necessary (only removing of the uniqs)